tests/Codeception/drone-system-run.sh

```diff
@@ -20,7 +20,7 @@ google-chrome --version
 echo "[RUNNER] Prepare Selenium"
 mkdir -p tests/Codeception/_output
 if [[ -f "/usr/lib/node_modules/selenium-standalone/lib/default-config.js" ]]; then
-	cp /usr/lib/node_modules/selenium-standalone/lib/default-config.js tests/Codeception/_output/selenium.config.js
+	flock tests/Codeception/_output/selenium.config.js cp /usr/lib/node_modules/selenium-standalone/lib/default-config.js tests/Codeception/_output/selenium.config.js
```

Comment:
```suggestion
	flock tests/Codeception/_output/selenium.config.js
	cp /usr/lib/node_modules/selenium-standalone/lib/default-config.js tests/Codeception/_output/selenium.config.js
```
Shouldn't the 2 commands (flock and cp) be separated either by a semicolon or a new line?

Reply:
nope: https://linux.die.net/man/1/flock
> [sic] The first and second forms wraps the lock around the executing a command, in a manner similar to [su](https://linux.die.net/man/1/su)(1) or [newgrp](https://linux.die.net/man/1/newgrp)(1).

Reply:
I see. Thanks for reporting back.